pkg/envoy/lds/connection_manager.go

```diff
@@ -17,13 +17,22 @@ import (
 	"github.com/openservicemesh/osm/pkg/featureflags"
 )
 
+// connectionDirection defines, for filter terms, the direction of the traffic from an application
+// perspective, in which the connection manager filters will be applied
+type connectionDirection string
```

Comment:
since the comment refers to the direction of traffic, maybe just call this `trafficDirection`.

Reply:
:+1: